cozepy/util.py

```
@@ -63,6 +63,18 @@
     return {k: v for k, v in d.items() if v is not None}
 
 
+def dump_dict_exclude_none(d: dict) -> dict:
+    res = {}
+    for k, v in d.items():
+        if v is None:
+            continue
+        elif isinstance(v, BaseModel):
+            res[k] = v.model_dump(exclude_none=True)
+        else:
+            res[k] = v
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "continue", remove the leading "el" from "elif"

(R1724)